lib/propcheck/utils.ex

```diff
@@ -35,9 +37,11 @@ defmodule PropCheck.Utils do
 
   # Check if verbose should be enabled
   def verbose?(opts) do
-    verbose_index = Enum.find_index(opts, & &1 == :verbose)
+    verbose_index = Enum.find_index(opts, &(&1 == :verbose))
+
     if verbose_index do
```

Comment:
If your are working on this, then please do me a favour with `if verbose_index != nil do`

Reply:
:+1:

Reply:
560c6bb

If we don't merge this PR here, we should cherry-pick that commit.